examples/nlp/gpt/conf/gpt_dpo.yaml

```diff
@@ -61,6 +61,12 @@ model:
     log_prob_forward_micro_batch_size: 1
     ref_policy_kl_penalty: 0.2
     average_log_probs: False
+    default_chosen_reward: 1.
+    default_rejected_reward: 0.
+    gt_reward_scale: 1.
+    preference_loss: dpo
```

Comment:
can you add a comment here on the other algorithms we support?

Reply:
Done